.gitignore

```
@@ -8,5 +8,6 @@
 app/release/
 Random-Name-Picker.iml
 *PremiumBackDoor.java
+*PremiumBackDoor.kt
```

This is more scalable and less lines.
```suggestion
*PremiumBackDoor*
```